### DISCUSSION TITLE
Need help - rawhidData was not declared in this scope.

### DISCUSSION THREAD

**Discussion by jackfruh** [NicoHood/HID#437] [Q&A]
I'm using a project that uses HID-Project.
Original at https://github.com/travisolbrich/Arduino-Emulated-Raildriver

It throws an error in the setup function.
rawhidData not declared.

I don't see where it's declared in the setup function (see below)

Does anyone know if it should be coming in via HID-Project.h?
I added HID-Project 2.8.4 in the Arduino IDE (2.2.1) via the library manager.

Here's a code snippet that's failing:
```
#include <Arduino.h>
#include "HID-Project.h"
#include "controller.cpp"

...

void setup()
{
    RawHID.begin(rawhidData, sizeof(rawhidData));
    Serial.begin(9600);
    
    for(int i = 0; i < sizeof(writeBuffer); i++)
    {
      writeBuffer[i] = 0;
    }

    writeBuffer[14] = 210;
}

Thanks!

**Comment by sjstein** [comment]
I believe this was solved - turns out the source this was grabbed from had a typo and did not declare the variable `rawhidData `before the method call.
